src/game.js

```diff
@@ -20,10 +20,11 @@ class Game {
       let id = Math.ceil(Math.random() * this.data.surveys.length);
       if (this.surveys.indexOf(id) === -1) this.surveys.push(id);
     }
+
     this.surveys = this.surveys.map(idNum => {
       let question = this.data.surveys.find(survey => survey.id === idNum);
       let answers =  this.data.answers.filter(answer => (answer.surveyId === idNum));
-      return ({question: question.question, answers: answers});
+      return ({'question': question.question, answers: answers});
```

Comment:
Why are you wrapping one key in a string? Could cause some weird behavior with accessing the property later on